tsconfig.json

```diff
@@ -2,15 +2,15 @@
   "exclude": ["./e2e"],
   "include": ["remix.env.d.ts", "**/*.ts", "**/*.tsx"],
   "compilerOptions": {
-    "lib": ["DOM", "DOM.Iterable", "ES2019"],
+    "lib": ["DOM", "DOM.Iterable", "ES2021"],
```

Comment:
Updating so I could use `replaceAll`. I know I don't necessarily need it, but it's a bit safer and a nice tool to have. No harm staying up to date.